assets-styleguide/js/styleguide.js

```diff
@@ -114,6 +121,17 @@ $('.sidenav').on('click', 'a', function(e) {
       } else if (window.location.hash !== newHash) {
         window.location.hash = newHash;
       }
+    },
+    done: function () {
```

Comment:
This seems potentially problematic if the focus happens after an animation is done. If a user is using the keyboard to navigation because they're not browsing the site visually, then this pause before the focus could be confusing. Although technically, I've never heard any official guidance on this. What do you think?